automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterConcurrencySpec.scala

```diff
@@ -111,6 +111,17 @@ class ClusterConcurrencySpec extends FreeSpec with LeonardoTestUtils with Parall
         }
       }
     }
+
+    // make sure defaultClientId works
+    "should set a default client id if specified" in {
+      withProject { project => implicit token =>
+        val request = defaultClusterRequest.copy(defaultClientId = Some("this is a client ID"))
+        withNewCluster(project, request = request) { cluster =>
```

Comment:
can you have param `monitorDelete=false`?

Reply:
ok!